jobs/fio-basic-local-disk.yaml

```diff
@@ -32,5 +32,6 @@ fio-setup-basic:
     - 1
     - 32
   test_size: 1g
+  testfile: y
```

Comment:
this patch need be merged with early patch, to make each commit of the series works as expected.